### PR TITLE
adds page for app launcher example in vertical navigation

### PR DESCRIPTION
**Pull request by matthewcarleton**
## Description

This adds an example of what the application launcher would look like with the vertical navigation.

The one thing to note is with the vertical nav example I did not include variations of the app launcher. This didn't seem necessary considering the layout required with pages showing vertical navigation examples.

It is related to the original work done for the app launcher here:
https://patternfly.atlassian.net/browse/PTNFLY-2372

## Changes

I've created horizontal and vertical specific pages where as before it was defaulted to horizontal but was not specifically named that way.

## Link to rawgit and/or image

https://rawgit.com/matthewcarleton/patternfly/app-launcher-vertical-dist/dist/tests/application-launcher-nav-vertical.html

## PR checklist (if relevant)

- [x] **Cross browser**: works in IE9
- [x] **Cross browser**: works in IE10
- [x] **Cross browser**: works in IE11
- [x] **Cross browser**: works in Edge
- [x] **Cross browser**: works in Chrome
- [x] **Cross browser**: works in Firefox
- [x] **Cross browser**: works in Safari
- [x] **Cross browser**: works in Opera
- [x] **Responsive**: works in extra small, small, medium and large view ports.
- [x] **Preview the PR**: An image or a URL for designer to preview this PR is provided.

### CODE REVIEW

**Comment by jeff-phillips-18** [comment]
I would have expected the https://rawgit.com/matthewcarleton/patternfly/app-launcher-vertical-dist/dist/tests/application-launcher-nav-vertical.html  page to look more like the https://rawgit.com/matthewcarleton/patternfly/app-launcher-vertical-dist/dist/tests/application-launcher-nav-horizontal.html page with the differing variations of the launcher.

**Comment by matthewcarleton** [comment]
@jeff-phillips-18 I thought about that but to show the variations of the app launcher it would create a page that was very long, and the differences are minor. I wanted to include the actual navigation here to show what it looks like on mobile as well as desktop.

File: src/less/application-launcher.less [hunk 1]
```diff
@@ -98,17 +98,19 @@
...
 
-
-.navbar-pf, .navbar-pf-alt {
-
-  .navbar-utility .applauncher-pf, .applauncher-pf {
+  .navbar-utility .applauncher-pf {
     .dropdown-menu {
       border-width: @applauncher-pf-menu-link-border-width !important;
       @media (min-width: @screen-sm-min) {
               margin-top: 11px;
```
Comment (jeff-phillips-18) [inline src/less/application-launcher.less]: the 11px to margin seems too much. The arrow is too separated from the navbar icon. Removing it and leaving as 0px looks better to me.

File: src/less/application-launcher.less [hunk 2]
```diff
@@ -98,17 +98,19 @@
...
             }
-
     }//.dropdown-menu
+}
+
+.navbar-pf, .navbar-pf-alt {
+
+  .navbar-utility .applauncher-pf, .applauncher-pf {
 
     &.open > a,
```
Comment (jeff-phillips-18) [inline src/less/application-launcher.less:115]: Are we supporting this if navbar-pf-alt? Is it viewable anywhere?

File: src/less/application-launcher.less (hunk 1 above)
Comment (matthewcarleton) [reply src/less/application-launcher.less]: I could change this, but it was inherited from whoever styled it originally.

Comment (jeff-phillips-18) [reply src/less/application-launcher.less]: Yeah, not due to this PR but it shows that the original setting is off.

@andresgalante @LHinson thoughts?

**Comment by jeff-phillips-18** [comment]
@matthewcarleton Yeah, showing the mobile mode is nice, I missed that it was doing that.

Maybe there should be a Application Launcher under widgets that shows off the various styles and have it show it for both vertical and horizontal nav, then change the Application Launcher for Horizontal Navigation page to also be full page. Just looking for consistency. This would be a follow-on PR, not for this PR.

File: src/less/application-launcher.less (hunk 1 above)
Comment (jeff-phillips-18) [reply src/less/application-launcher.less]: @beanh66 @serenamarie125 thoughts?

**Comment by serenamarie125** [comment]
@matthewcarleton the tooltip seems to be pretty low, doesn't seem to match what's here on the design page http://www.patternfly.org/pattern-library/application-framework/launcher/#/design
@beanh66, @jennyhaines please take a look

**Comment by matthewcarleton** [comment]
@serenamarie125 @jeff-phillips-18 @beanh66 ya agreed, I am happy to make that change in this PR.

**Comment by jennyhaines** [comment]
@matthewcarleton @serenamarie125 - the top border on the tooltip should sit 3px below the bottom of the masthead, to be consistent with PatternFly. Like this: 
mockup

**Comment by matthewcarleton** [comment]
@jennyhaines ok, thanks!

**Review by jeff-phillips-18 — APPROVED** [review]
LGTM! Thanks Matt.

**Review by cdcabrera — APPROVED** [review]
Looks good.

To bad we can't adjust the "These examples are for development testing purposes..." notification

**Comment by cdcabrera** [comment]
@matthewcarleton ~~can you take a look at Travis failing on your PR~~

**Comment by jeff-phillips-18** [comment]
@cdcabrera it is due to the semantic release issues currently being worked.

**Comment by cdcabrera** [comment]
@matthewcarleton @jeff-phillips-18  is spot on...  

Did a quick rebase test on the files involved looks like you'll potentially have a merge conflict around "npm shrinkwrap", once the semantic release commits get worked through

**Comment by matthewcarleton** [comment]
Ok, this is good to be merged :)

**Review by andresgalante — APPROVED** [review]
Thanks @matthewcarleton great work!

@jeff-phillips-18 Do we need to change the commit message for the new automatic release or is this one ok to merge? If it is, please go ahead an merge it

**Comment by jeff-phillips-18** [comment]
The commit message should be updated as such:

fix(applauncher): adds page for app launcher example in vertical navigation

@matthewcarleton let me know if you need help adjusting the commit wording.

**Comment by matthewcarleton** [comment]
@jeff-phillips-18 ok, I've updated the commit message :)

**Comment by jeff-phillips-18** [comment]
Could you squash the commits please?

**Pull request merged**